### PR TITLE
Add OpEvo example

### PR DESCRIPTION
**Pull request by gxiaotian**
This pr aims to provide examples of tuning tensor operators with NNI. Three tuning algorithms, OpEvo, G-BFS and N-A2C are implemented with NNI. All examples locate at `examples/trials/systems/opevo/`, and a doc file `docs/en_US/TrialExample/OpEvoExamples.md` is also provided for demonstrating how to set up experiment environment and run these experiments. Please refer to the doc for detailed explanation of this pr.

### CODE REVIEW

**Comment by QuanluZhang** [comment]
@gxiaotian could you write description of this pr, which makes it easy for others to review

**Comment by gxiaotian** [comment]
@QuanluZhang Description added. Would like to add more if needed. Thanks.

File: README_zh_CN.md [hunk 1]
```diff
@@ -356,4 +356,4 @@ You can use these commands to get more information about the experiment
 
 ## **许可协议**
 
-代码库遵循 [MIT 许可协议](LICENSE)
\ No newline at end of file
+代码库遵循 [MIT 许可协议](LICENSE)
```
Comment (QuanluZhang) [inline README_zh_CN.md:359]: please revert the changes to chinese doc

File: docs/en_US/TrialExample/OpEvoExamples.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Tensor operator optimization is substantially a combinatorial optimization problem. The objective function is the performance of a tensor operator on specific hardware platform, which should be maximized with respect to the hyper-parameters of corresponding device code, such as how to tile a matrix or whether to unroll a loop. This example illustrates how to automatically tune tensor operators with NNI. Three tuning algorithms, OpEvo, G-BFS and N-A2C are provided. Please refer to [OpEvo: An Evolutionary Method for Tensor Operator Optimization](https://128.84.21.199/abs/2006.05664) for detailed explanation about these algorithms.
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/OpEvoExamples.md:7]: could you use arxiv link for the paper, not the IP one?

File: docs/en_US/TrialExample/OpEvoExamples.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+```bash
+# if you are using Nvidia GPU
+make cuda-env
+# if you are using AMD GPU
+make rocm-env
+```
+
+## Run Experiments:
+
+Three representative kinds of tensor operators, matrix multiplication, batched matrix multiplication and 2D convolution, are chosen from BERT and AlexNet, and tuned with NNI. The `Trial` code for all tensor operators is `/root/compiler_auto_tune_stable.py`, and `Search Space` files and `config` files for each tuning algorithm locate in `/root/experiments/`, which are categorized by tensor operators. Here `/root` refers to the root of the container.
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/OpEvoExamples.md:22]: "matrix multiplication, batched matrix multiplication and 2D convolution" -> "**matrix multiplication**, **batched matrix multiplication** and **2D convolution**"

File: README_zh_CN.md (hunk 1 above)
Comment (gxiaotian) [reply README_zh_CN.md:359]: I didn't change this file in fact. I believe there might be some invisible characters modified by the editor. I will try to figure it out.

File: docs/en_US/TrialExample/OpEvoExamples.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+For tuning the operators of matrix multiplication, please run below commands from `/root`:
```
Comment (QuanluZhang) [inline docs/en_US/TrialExample/OpEvoExamples.md:24]: how to start the container?

Comment (QuanluZhang) [reply docs/en_US/TrialExample/OpEvoExamples.md:24]: got it. `make cuda-env` will directly start the container, right?

**Comment by QuanluZhang** [comment]
looks great to me.

File: docs/en_US/TrialExample/OpEvoExamples.md (hunk 4 above)
Comment (gxiaotian) [reply docs/en_US/TrialExample/OpEvoExamples.md:24]: Yes, you are right~

File: README_zh_CN.md (hunk 1 above)
Comment (gxiaotian) [reply README_zh_CN.md]: updated

File: docs/en_US/TrialExample/OpEvoExamples.md (hunk 2 above)
Comment (gxiaotian) [reply docs/en_US/TrialExample/OpEvoExamples.md]: updated

File: docs/en_US/TrialExample/OpEvoExamples.md (hunk 3 above)
Comment (gxiaotian) [reply docs/en_US/TrialExample/OpEvoExamples.md]: updated
